styles.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  width: 100%;
+  max-width: 375px;
+  font-family: 'poppins', sans-serif;
+}
+
+.background-img {
+  background-image: url(./Images/header-bg.png);
+  background-repeat: no-repeat;
+  height: 680px;
+}
+
+.header-flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.logo-link {
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  padding: 10px 0 0 10px;
+}
+
+.union-icon {
+  width: 18px;
+  height: 12px;
+  margin-right: 10px;
+  cursor: pointer;
+}
```

Comment:
- [ ] Kindly make sure you are using the correct spacing value at the `right` of the **Hamburger** button as the one used on your chosen Figma template.